<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Perspective</class>
 <widget class="QDialog" name="Perspective">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>透视变换</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/icon/transform/Image/stock-trans-perspective.png</normaloff>:/icon/transform/Image/stock-trans-perspective.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>坐标</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>X</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Y</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>左上角</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_9">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_TL_x">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_10">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_TL_y">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>左下角</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_3">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_BL_x">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_4">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>20</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_BL_y">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>右上角</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_5">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>20</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_TR_x">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_7">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_TR_y">
       <property name="minimum">
        <number>-100</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>右下角</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_6">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>20</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_BR_x">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_8">
       <property name="minimumSize">
        <size>
         <width>52</width>
         <height>20</height>
        </size>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSlider_BR_y">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="pushButton_reset">
       <property name="text">
        <string>重置</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_OK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider_BL_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_3</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>76</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_3</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_BL_x</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_4</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_BL_y</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_BL_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_4</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_TR_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_5</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>78</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_5</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_TR_x</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>67</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>118</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_7</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_TR_y</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_TR_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_7</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>277</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_6</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_BR_x</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>137</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_BR_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_6</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_8</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_BR_y</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_BR_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_8</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_9</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_TL_x</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>41</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_TL_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_9</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>121</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>85</x>
     <y>53</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_10</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider_TL_y</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_TL_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_10</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>338</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>285</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_OK</sender>
   <signal>clicked()</signal>
   <receiver>Perspective</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>219</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>Perspective</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
